utils/build-script-impl

```diff
@@ -2623,12 +2637,14 @@ for host in "${ALL_HOSTS[@]}"; do
                   echo "Cleaning the XCTest build directory"
                   call rm -rf "${XCTEST_BUILD_DIR}"
 
+                  LLVM_BIN="$(build_directory_bin ${LOCAL_HOST} llvm)"
```

Comment:
Where is this new variable used?

Reply:
In the options immediately below, e.g. Line 2645.  Previously it depended on this being set somewhere else